jest.config.js

```diff
@@ -0,0 +1,14 @@
+const merge = require('merge');
+const tsPreset = require('ts-jest/jest-preset');
+
+module.exports = merge.recursive(tsPreset, {
```

Comment:
Manually merging our config with the preset, is that necessary? Looking at the configuration docs for ts-jest, the `preset` key can be used.<sup>[\[1\]][1]</sup> Merging this manually also complicates this process as we both now have an extra dependency and a resolution process (i.e. how does `merge.recursive` work? how many levels does it do? how does it treat the arrays?) to manage.

If we can get away with using `preset: 'ts-jest'`, let use that.

  [1]:https://kulshekhar.github.io/ts-jest/user/config/#basic-usage

Reply:
this came from an earlier point where there were two presets, so i had to merge the presets manually. not needed any more :+1: